docs/modules/ROOT/pages/documentation/ecosystem/maven-plugin.adoc

```diff
@@ -317,6 +320,43 @@ the process id of the running Payara Micro instance.
 
 |===
 
+=== `reload`
+*Aim*: This goal reloads the Payara Micro application with the specified configuration.
+By default, this goal performs the vanilla deployment of the application without restarting the 
+Payara Micro instance. If `hotDeploy` is set to true then the Hot Reload functionality
```

Comment:
Again, should this say Hot Deploy instead of Hot Reload? The note below also mentions Hot Deploy.

Reply:
By default this goal does not perform the Hot Deploy function and only updates the `.reload` so named it `reload` task.

Reply:
Yes, the name of the task is OK. But I'm referring to the "Hot Reload" in the sentence above. I think it should be:
```suggestion
Payara Micro instance. If `hotDeploy` is set to true then the Hot Deploy functionality
```

Reply:
Thanks @OndroMih